runtime/gc_structs/ClassIterator.hpp

```diff
@@ -94,13 +96,18 @@ class GC_ClassIterator {
 		, _callSitesIterator(clazz)
 		, _methodTypesIterator(clazz->romClass->methodTypeCount, clazz->methodTypes)
 		, _varHandlesMethodTypesIterator(clazz->romClass->varHandleMethodTypeCount, clazz->varHandleMethodTypes)
+		, _shouldScanClassObject(true)
 	{}
 
 	MMINLINE int getState() 
 	{ 
 		return _state; 
 	}
 
+	MMINLINE bool ShouldScanClassObject() 
```

Comment:
I believe we should not add function which would not be used today or nearest future

Reply:
👍